index.html

```diff
@@ -25,8 +15,16 @@
         rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script src="https://use.fontawesome.com/303b1d319c.js"></script>
-    
- 
+    <script>
+        window.dataLayer = window.dataLayer || [];
+
+        function gtag() {
+            dataLayer.push(arguments);
+        }
+        gtag('js', new Date());
+
+        gtag('config', 'G-MZLCHLPPZL');
```

Comment:
- [ ] Keep all the JavaScript code in JS files, not in the script tag as per the [JavaScript requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)